geoportal/c2cgeoportal_geoportal/scaffolds/update/{{cookiecutter.project}}/CONST_CHANGELOG.txt

```diff
@@ -28,6 +28,20 @@ Information
    ``service: geoportal`` by ``service: geoportal-advance``, and in service ``alembic`` you should replace
    ``service: alembic`` by ``service: alembic-advance``.
 
+4. Some little changes in the application build:
+   - We no longer use custom build commands to build the docker images, we simply use the docker-compose
```

Comment:
- In ./build script, we no longer call `docker build` command with custom parameters, but call `docker-compose build` command with wo additional environment variables: